src/olympia/devhub/templates/devhub/validate_addon.html

```diff
@@ -42,21 +42,14 @@ <h2>{{ title }}</h2>
   <section id="upload-file" class="validate-addon">
     <div class="list-addon">
       <label>{{ _('Do you want your add-on to be distributed on this site?') }}</label>
-      <div>
-        <label for="{{ new_addon_form.is_unlisted.auto_id }}">
-          {{ new_addon_form.is_unlisted }}
-          {{ new_addon_form.is_unlisted.label }}
-          <span class="tip tooltip"
-                title="{{ new_addon_form.is_unlisted.help_text }}">?</span>
-        </label>
-      </div>
+      {{ new_addon_form.channel }}
     </div>
 
     <input type="file" id="upload-addon"
       data-upload-url="{{ url('devhub.standalone_upload') }}"
       data-upload-url-listed="{{ url('devhub.standalone_upload') }}"
       data-upload-url-unlisted="{{ url('devhub.standalone_upload_unlisted') }}"">
-
+      
```

Comment:
Remove this extra whitespace on the line

Reply:
@eviljeff I have removed the spaces and Sorry for the amend this one more time. I will definitely take care in the future. :P